alacritty/src/display/mod.rs

```diff
@@ -1529,6 +1550,54 @@ impl<T> DerefMut for Replaceable<T> {
     }
 }
 
+/// The frame timer state.
+#[derive(Default)]
+pub struct FrameTimerInfo {
+    /// The optimal delay for the renderer.
+    swap_delay: Option<Duration>,
+
+    /// The time stamp of the previous contiguos frame.
```

Comment:
```suggestion
    /// The timestamp of the previous contiguous frame.
```

Besides the typo, what is "contiguous frame" supposed to mean here? Isn't it already implied by "previous" that we're talking about the frame right before this?

I think just this should be better:

```suggestion
    /// The timestamp of the previous frame.
```